Release/src/websockets/client/ws_client_wspp.cpp

```diff
@@ -239,8 +239,8 @@ class wspp_callback_client : public websocket_client_callback_impl, public std::
                     }
                     else
                     {
-                        const auto &server_name = utility::conversions::to_utf8string(m_uri.host());
```

Comment:
Using `const auto&` here avoids a copy when the native string type is `std::string`.